ledger/state.go

```diff
@@ -712,8 +712,9 @@ func (ls *LedgerState) ledgerProcessBlocks() {
 							cutoffSlot = 0
 						}
 
-						// Validate only if blockSlot ≥ cutoffSlot and skip if it fails
-						if blockSlot >= cutoffSlot {
+						// Validate only if ValidateHistorical is enabled and blockSlot ≥ cutoffSlot
+						if ls.config.ValidateHistorical &&
+							blockSlot >= cutoffSlot {
```

Comment:
Is this going to work to validate near current tip if ValidateHistorical is disabled?